fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/directory/FDBDirectory.java

```diff
@@ -350,7 +373,14 @@ public void deleteFile(@Nonnull String name) throws IOException {
     @Override
     public long fileLength(@Nonnull String name) throws NoSuchFileException {
         LOGGER.trace("fileLength -> {}", name);
+        name = convertToDataFile(name);
         FDBLuceneFileReference reference = context.asyncToSync(FDBStoreTimer.Waits.WAIT_LUCENE_FILE_LENGTH, getFDBLuceneFileReference(name));
+        if (name.endsWith(".cfe")) {
```

Comment:
There seem to be a lot of places where these are hardcoded. If we are going to use term vectors I believe we'll want to add those into the compressed reference as well.